pokedex/data/csv/items.csv

```diff
@@ -303,98 +303,98 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
 302,reaper-cloth,10,2000,10,
 303,razor-claw,12,5000,80,
 304,razor-fang,12,5000,30,7
-305,tm01,37,5000,,
-306,tm02,37,3000,,
-307,tm03,37,3000,,
-308,tm04,37,3000,,
-309,tm05,37,3000,,
-310,tm06,37,3000,,
-311,tm07,37,3000,,
-312,tm08,37,3000,,
-313,tm09,37,3000,,
-314,tm10,37,10000,,
-315,tm11,37,3000,,
-316,tm12,37,5000,,
+305,tm01,37,40000,,
+306,tm02,37,1000,,
+307,tm03,37,50000,,
+308,tm04,37,50000,,
+309,tm05,37,50000,,
+310,tm06,37,1000,,
+311,tm07,37,1000,,
+312,tm08,37,50000,,
+313,tm09,37,50000,,
+314,tm10,37,1000,,
+315,tm11,37,1000,,
+316,tm12,37,50000,,
 317,tm13,37,10000,,
-318,tm14,37,10000,,
-319,tm15,37,3000,,
-320,tm16,37,3000,,
-321,tm17,37,20000,,
-322,tm18,37,20000,,
-323,tm19,37,50000,,
-324,tm20,37,10000,,
-325,tm21,37,10000,,
-326,tm22,37,10000,,
+318,tm14,37,1000,,
+319,tm15,37,1000,,
+320,tm16,37,1000,,
+321,tm17,37,10000,,
+322,tm18,37,10000,,
+323,tm19,37,10000,,
+324,tm20,37,34464,,
```

Comment:
34464 is an... oddly specific number. Bulbapedia says TM20 is actually 100,000 pokedollars. i guess cost isn't a uint16 any more.

Reply:
Nice catch, i completely missed this. I'll try to fix it in the next couple of days

Reply:
Bingo, it's now uint32. Only 7 items (tm20 and 6 other tms) need the third byte (so far nothing needs the fourth byte - and probably never will, i guess).